src/go/plugin/go.d/config/go.d/snmp.profiles/default/_cisco-asa.yaml

```diff
@@ -76,7 +84,8 @@ metrics:
       - OID: 1.3.6.1.2.1.99.1.1.1.4
         name: entPhySensorValue
         description: Most recent measurement obtained by the agent for this sensor
-        unit: "TBD"
+        unit: "1"
```

Comment:
This is the same kind of thing as above. Where the other is the Cisco derivative, this property (and surrounding object) is the standard way to express such generic sensors, as described in [RFC 3433](https://www.rfc-editor.org/rfc/rfc3433.html).

Reply:
I'm fine with leaving these generic sensors for now and represent them with unit `1`, but we need to address that eventually, with a dedicated flag.